plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -69,6 +70,9 @@ function perflab_query_plugin_info( string $plugin_slug ) {
 
 	$plugins = array();
 	foreach ( $response->plugins as $plugin_data ) {
+		if ( ! in_array( $plugin_data['slug'], perflab_get_standalone_plugins(), true ) ) {
```

Comment:
More efficient would be to move the call outside the loop and store the plugins in a variable.

If you do that, you could even add another little performance enhancement: You could use `array_flip()` on the plugin list, and then use `isset()` instead of `in_array()` (as `isset()` is faster).

Reply:
Done in https://github.com/WordPress/performance/pull/1573/commits/dc9f0e349a430b029141895784ad9e16222b9b2a